DFC.ServiceTaxonomy.GraphSync/GraphSyncers/Helpers/EmbeddedContentItemsGraphSyncer.cs

```diff
@@ -456,5 +456,24 @@ private ContentItem[] ConvertToContentItems(JArray? contentItems)
                 throw new GraphSyncException("Embedded content container does not contain ContentItems.");
             return embeddedContentItems;
         }
+
+        public async Task AddRelationship(IDescribeRelationshipsContext context)
+        {
+            var items = (JArray)context.ContentItem.Content["FlowPart"]["Widgets"];
```

Comment:
This is a generic embedded content items helper and shouldn't have part specific code in it.
If you need code specific for a particular part syncer, there are derived versions of this class that the logic can live in.